src/asl.ts

```diff
@@ -1060,6 +1076,32 @@ export class ASL {
           },
         };
       });
+    } else if (isDebuggerStmt(stmt) || isEmptyStmt(stmt)) {
+      return {
+        startState: "pass",
+        states: {
+          pass: {
+            Type: "Pass",
+          },
+        },
+      };
```

Comment:
nit: you can return `undefined` or `{ Type: "Pass" }`

Reply:
`undefined` seems beter?